src/linker.ts

```diff
@@ -94,24 +105,25 @@ const scopeWithin = (includeInheritedMembers: boolean) => (node: Node<Linked>):
 const assignScopes = (environment: Environment<Linked>) => {
   const globalScope = assign({},
     ...environment.children().map(scopeContribution),
-    ...environment.getNodeByFQN<Package>('wollok.lang').children().map(scopeContribution),
-    ...environment.getNodeByFQN<Package>('wollok.lib').children().map(scopeContribution)
+    ...environment.getNodeByFQN<Package>('wollok.lang').members.map(scopeContribution),
+    ...environment.getNodeByFQN<Package>('wollok.lib').members.map(scopeContribution)
```

Comment:
I imagine a possible refactor: having all core files in a list and generate globalScope from that list